src/Engine/ProtoCore/Utils/CoreUtils.cs

```diff
@@ -7,6 +7,9 @@ namespace ProtoCore.Utils
 {
     public static class CoreUtils
     {
+        public static readonly string NodeFunctionClassName = "Function";        
+        public static readonly string NodeLibraryMirronFunctionObjectName = "FunctionObject.ds";
```

Comment:
I would name them `FunctionObjectClass` and `FunctionObjectLibrary`.

Reply:
Changing in the next commit... Thanks!